rel-eng/releasers.conf

```diff
@@ -1,33 +1,33 @@
 [koji-katello]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora23 katello-nightly-fedora22 katello-thirdparty-rhel7
+autobuild_tags = katello-3.3-rhel5 katello-3.3-rhel6 katello-3.3-rhel7 katello-3.3-fedora24 katello-3.3-fedora25 katello-3.3-thirdparty-rhel7
 
 [koji-katello-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora23 katello-nightly-fedora22 katello-thirdparty-rhel6 katello-thirdparty-rhel7
+autobuild_tags = katello-3.3-rhel5 katello-3.3-rhel6 katello-3.3-rhel7 katello-3.3-fedora23 katello-3.3-fedora22 katello-3.3-thirdparty-rhel6 katello-3.3-thirdparty-rhel7
```

Comment:
The fedora tags here are wrong, which this releaser isn't used for releases, but could avoid confusion.